README.md

```diff
@@ -341,6 +341,7 @@ Additional lists you might find useful:
 
 - [Captcha plugin](https://github.com/dereuromark/cakephp-captcha) - Simple, unobstrusive and extendable captcha solution providing by default an image based math captcha.
 - [CipherBehavior plugin](https://github.com/adayth/cakephp-cipher-behavior) - Cipher your entities data with this behavior. Encryption is done at PHP level using CakePHP Security class.
+- [Bcrowe/EncryptedType](https://github.com/bcrowe/cakephp-encrypted-type) - Simple solution that enables Encrypted data types to be stored in the database.
```

Comment:
The order seems off.

Reply:
Plugin name is wrong

Reply:
Renamed to Bcrowe/CakephpEncryptedType and fixed order

Reply:
The package name is bcrowe/cakephp-encrypted-type.
Your plugin is actually named https://github.com/bcrowe/cakephp-encrypted-type/blob/master/composer.json#L27
`BryanCrowe/EncryptedType`, as per CakePHP plugin documentation.

Ideally the name is the same as the package name though to avoid confusion.

Reply:
Ahh ok, I wasn't sure what name to put there.

Reply:
I clarified in https://github.com/FriendsOfCake/awesome-cakephp/blob/master/CONTRIBUTING.md#tips-for-creating-composer-packages